scripts/irods/test/test_chunkydevtest.py

```diff
@@ -698,9 +698,9 @@ def test_large_files_from_devtest(self):
             os.unlink(lrsfile)
         if os.path.isfile(rsfile):
             os.unlink(rsfile)
-        self.admin.assert_icommand("iput -vbPKr --retries 10 --wlock -X " + rsfile + " --lfrestart " +
+        self.admin.assert_icommand("iput -vbPkKr --retries 10 --wlock -X " + rsfile + " --lfrestart " +
```

Comment:
adding `-k` fixed this assertion. does this change the intention of the test in any way?

Reply:
It does not - the assertion here is about using the restart file, so I think you're good to go.

Reply:
So, what is the significance of the ichksum calls/options? Are they needed at all for testing restarts?

Reply:
I would think not.  I would speculate these are here for coverage more than targeted testing.

I'd be okay if you removed them here.

Reply:
ok. i'll look into removing the checksum options on iput as well as the ichksum calls.